config/config_test.go

```diff
@@ -3,35 +3,137 @@
 
 package config
 
-/*
-func TestAddDomainsDuplicate(t *testing.T) {
-	domains := []string{"twitter.com", "google.com", "twitter.com"}
+import (
+	"net"
+	"reflect"
+	"sort"
+	"testing"
+)
 
-	e := NewEnumeration()
-	e.Config.AddDomains(domains)
+func TestCHeckSettings(t *testing.T) {
+	c := NewConfig()
 
-	got := e.Config.Domains()
-	want := append(domains[0:2])
+	err := c.CheckSettings()
 
-	if !reflect.DeepEqual(got, want) {
-		t.Errorf("mismatched result, got %+v, want %+v", got, want)
+	if err != nil {
+		t.Errorf("Error checking settings.\n%v", err)
+	}
+
+}
+func TestDomainRegex(t *testing.T) {
+	c := NewConfig()
+	got := c.DomainRegex("owasp.org")
+
+	if got != nil {
+		t.Errorf("Error with DomainRegex.\n%v", got)
 	}
 }
 
-func TestIsDomainInScope(t *testing.T) {
-	domain := "google.com"
-	inScope := "mail.google.com"
-	outOfScope := "mail.random.com"
+func TestAddDomains(t *testing.T) {
+	c := NewConfig()
+	example := "owasp.org/test"
```

Comment:
This isn't something that should be accepted by `AddDomains`. Looks like we should be performing better input validation since it seems to work just fine...

We should address this when we go to refactor the config package.